test-clients/src/main/resource/spec-default.properties

```diff
@@ -14,8 +14,9 @@ app.properties.defaultsPath=dev-system-files/appProperties.bin
 #ci.properties.map=tps=500,mins=1,burstSize=100,allowedSecsBelow=10
 ### UmbrellaRedux ###
 #ci.properties.map=duration=60,unit=SECONDS,maxOpsPerSec=10,props=regression-file_ops.properties,maxPendingOps=50
-ci.properties.map=tokensPerTxn=1,sendingAccountsPerToken=1,receivingAccountsPerToken=1,balanceInit=10000
+#ci.properties.map=tokensPerTxn=1,sendingAccountsPerToken=1,receivingAccountsPerToken=1,balanceInit=10000
 #ci.properties.map=numTokens=10,numAccounts=10
+ci.properties.map=totalTestTokenAccounts=100,totalTestTokens=10,testTreasureStartAccount=1001
```

Comment:
If it is modified for testing , it can be reverted back ?

Reply:
Reverted back even though it doesn't matter.